components/ExpenseInvoice.js

```diff
@@ -30,8 +30,10 @@ const chunkItems = (expense) => {
 
   // Estimate the space available
   const countLines = (str) => sumBy(str, (c) => c === '\n');
-  const billFromAddressSize = countLines(get(expense.account, 'location.address', ''));
-  const billToAddressSize = countLines(get(expense.payeeLocation, 'address', ''));
+  const billFromAddressSize = countLines(get(expense.payeeLocation, 'address', ''));
+  const billToAddressSize = countLines(
```

Comment:
We need to make sure addresses are loaded from the `billTo` account by default, with a fallback to the host if there's nothing set. Maybe worth introducing a `getBillToAddress`